examples/tcp-socket-listener-client/tcp_socket_client.bal

```diff
@@ -43,13 +55,8 @@ service ClientService = service {
         }
     }
 
-    // This is invoked once the connection is closed.
-    resource function onClose(socket:Caller caller) {
-        io:println("Leave from: ", caller.remotePort);
-    }
-
     // This resource is invoked for the error situation
-    // if it happens during the `onConnect`, `onReadReady`, and `onClose` functions.
+    // if it happens during the `onConnect` and `onReadReady` functions.
```

Comment:
This resource is invoked by the`onConnect` and `onReadReady` functions in erroneous situations.